config/profiles.json

```diff
@@ -1,6 +1,82 @@
 {
     "rollup": {
         "azure_sdk": [
+        {
+            "name": "V2019_03_01",
+            "resourceTypes": {
+                "Microsoft.Authorization": {
+                    "management": {
+                        "2015-07-01": ["*"]
+                    }
+                },
+                "Microsoft.Locks" : {
+                    "management": {
+                        "2016-09-01": ["*"]
+                    }
+                },
+                "Microsoft.Policy" : {
+                    "management": {
+                        "2016-12-01": ["*"]
+                    }
+                },
+                "Microsoft.Compute" : {
+                    "management": {
+                        "2017-03-30": ["*"],
+                        "2017-12-01": ["*"]
+                    }
+                },
+                "Microsoft.KeyVault" : {
+                    "management": {
+                        "2016-10-01": ["*"]
+                    },
+                    "data": {
+                        "2016-10-01": ["*"]
+                    }
+                },
+                "Microsoft.Network" : {
+                    "management": {
+                        "2017-10-01": ["*"]
+                    }
+                },
+                "Microsoft.Dns": {
+                    "management": {
+                        "2016-04-01": ["*"]
+                    }
+                },
+                "Microsoft.Resources" : {
+                    "management": {
+                        "2018-05-01": ["*"]
+                    }
+                },
+                "Microsoft.Subscriptions" : {
+                    "management": {
+                        "2016-06-01": ["*"]
+                    }
+                },
+                "Microsoft.Storage" : {
+                    "management": {
+                        "2017-10-01": ["*"]
+                    }
```

Comment:
I believe storage data plane still resides in a separate repo and its hand crafted..
We use the following packages for AZURESTACK at the moment, I will see whether we need to take any later versions
https://rubygems.org/gems/azure-storage-common/versions/1.0.1
Blob: https://rubygems.org/gems/azure-storage-blob/versions/1.0.1
Queue: https://rubygems.org/gems/azure-storage-queue/versions/1.0.1
Table: https://rubygems.org/gems/azure-storage-table/versions/1.0.1